core/src/test/scala/unit/kafka/server/DescribeLogDirsRequestTest.scala

```diff
@@ -46,15 +48,23 @@ class DescribeLogDirsRequestTest extends BaseRequestTest {
 
     val request = new DescribeLogDirsRequest.Builder(new DescribeLogDirsRequestData().setTopics(null)).build()
     val response = connectAndReceive[DescribeLogDirsResponse](request, destination = controllerSocketServer)
-    val logDirInfos = response.logDirInfos()
+    case class ReplicaInfo(size: Long, offsetLag: Long, isFuture: Boolean)
+    case class LogDirInfo(error: Errors, replicaInfos: Map[TopicPartition, ReplicaInfo])
```

Comment:
That's a pity that we have to redefine there classes here. Couldn't we update the test to work with the plain response instead?